### ISSUE TITLE
Color Splasher bug

### ISSUE THREAD

**Issue by Thomas84** [pyrevitlabs/pyRevit#2026]
# 🐞 Describe the bug

I have tested the Color Splasher tool. The category dropdown are in german, but they don't display umlauts (äöü). Selecting a category creates an error. 
The buttons at the bottom don't work at all
The button for Reset All Colors is too small

# ♻️ To Reproduce
1. Open Revit 2024.2 German version (English version works in general)
2. Golden nugget arch example project
3. Go to pyRevit Color Splasher
4. Select category wall (Wände)

Bug when using create view Filters (with 2024.2 English):
![image](https://github.com/eirannejad/pyRevit/assets/5624034/73547487-a535-4aa0-893f-9781ea524f1c)
Maybe you can disable the Create View Filters button, if a double category is selected?

# ⏲️ Expected behavior
no error

# 🖼 Screenshots of the issue
![image](https://github.com/eirannejad/pyRevit/assets/5624034/e629fddc-18b9-4225-8c60-411a1975cc7f)

```
************** Ausnahmetext **************
IronPython.Runtime.UnboundNameException: name 'DataTable' is not defined
   bei IronPython.Runtime.Operations.PythonOps.GetVariable(CodeContext context, String name, Boolean isGlobal, Boolean lightThrow)
   bei IronPython.Compiler.LookupGlobalInstruction.Run(InterpretedFrame frame)
   bei Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
   bei Microsoft.Scripting.Interpreter.LightLambda.Run4[T0,T1,T2,T3,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3)
   bei IronPython.Compiler.PythonCallTargets.OriginalCallTarget3(PythonFunction function, Object arg0, Object arg1, Object arg2)
   bei System.Dynamic.UpdateDelegates.UpdateAndExecute5[T0,T1,T2,T3,T4,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2, T3 arg3, T4 arg4)
   bei CallSite.Target(Closure , CallSite , Object , Object , EventArgs )
   bei System.Dynamic.UpdateDelegates.UpdateAndExecute3[T0,T1,T2,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2)
   bei _Scripting_(Object[] , Object , EventArgs )
   bei System.Windows.Forms.ComboBox.OnSelectedIndexChanged(EventArgs e)
   bei System.Windows.Forms.ComboBox.WmReflectCommand(Message& m)
   bei System.Windows.Forms.ComboBox.WndProc(Message& m)
   bei System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
```

![image](https://github.com/eirannejad/pyRevit/assets/5624034/8cc36f6f-6b15-44b7-80f0-94994b8fea47)

![image](https://github.com/eirannejad/pyRevit/assets/5624034/d2459cc9-5776-4931-9fd8-1f7576a74940)

# 🖥️ Hardware and Software Setup:
 - OS: win10
 - pyRevit Version 4.8.14.23353+1222

```
pyChilizer | Type: UIExtension | Repo: "" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\pyChilizer.extension"
PyRevitMEP | Type: UIExtension | Repo: "" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\PyRevitMEP.extension"
PyRevitPlus | Type: UIExtension | Repo: "https://github.com/gtalarico/pyrevitplus.git" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\PyRevitPlus.extension"
pyRevitTutor | Type: UIExtension | Repo: "" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\pyRevitTutor.extension"
pySSG | Type: UIExtension | Repo: "https://github.com/kbruxvoort/SSG.extension.git" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\pySSG.extension"
pyStructure | Type: UIExtension | Repo: "" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\pyStructure.extension"
pyTiBa | Type: UIExtension | Repo: "https://github.com/tillbaum/pytiba.git" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\pyTiBa.extension"
Revitron | Type: LibraryExtension | Repo: "https://github.com/revitron/revitron.git" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\Revitron.lib"
Revitron UI | Type: UIExtension | Repo: "https://github.com/revitron/revitron-ui.git" | Installed: "C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions\Revitron UI.extension"
==> Default Extension Search Path
C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions
==> Extension Search Paths
C:\Users\thomas.vogt\AppData\Roaming\pyRevit\Extensions
C:\Users\thomas.vogt\AppData\Roaming\pyRevitCustomExtensions
==> Extension Sources - Default
https://github.com/eirannejad/pyRevit/raw/master/extensions/extensions.json
==> Extension Sources - Additional
==> Installed Revits
Autodesk Revit 2024 | Version: 24.2.0.63 | Build: 20231029_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2024\"
Autodesk Revit 2023 | Version: 23.1.30.97 | Build: 20230828_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2023\"
Autodesk Revit 2022 | Version: 22.1.50.17 | Build: 20230915_1530(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2022\"
Autodesk Revit 2021 | Version: 21.1.90.15 | Build: 20230907_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2021\"
Autodesk Revit 2020 | Version: 20.2.90.12 | Build: 20220517_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2020\"
Autodesk Revit 2019 | Version: 19.2.60.3 | Build: 20220224_1700(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2019\"
2018.3.3 Security Fix | Version: 18.3.3.18 | Build: 20190510_1515(x64) | Language: 1033 | Path: "C:\Program Files\Autodesk\Revit 2018\"
```

**Comment by jmcouffin** [comment]
fixed by #2029 
for installers, in about 10min.
https://github.com/eirannejad/pyRevit/actions/runs/7273980353

Please try it and report here @Thomas84

**Comment by jmcouffin** [comment]
Works for Double with filter creation but not with floats 🤣 anymore...

**Comment by jmcouffin** [comment]
these installers in 10min
https://github.com/eirannejad/pyRevit/actions/runs/7274356871

**Comment by RubenSLT** [comment]
Good afternoon
I have the same problem, can you help me?.
I attach information on this subject
![image](https://github.com/pyrevitlabs/pyRevit/assets/174019445/00a3f186-d566-41f8-80cd-d443e606af1d)
![image](https://github.com/pyrevitlabs/pyRevit/assets/174019445/28fb64bf-acf3-4281-89f6-04515cb2fd41)
System.MissingMemberException: 'FormCats' object has no attribute '_listBox2'
   en IronPython.Runtime.Binding.MetaUserObject.FastGetBinderHelper.<>c__DisplayClass16_0.<FallbackError>b__1(CallSite site, Object self, CodeContext context)
   en IronPython.Runtime.Types.GetMemberDelegates.SlotDict(CallSite site, Object self, CodeContext context)
   en System.Dynamic.UpdateDelegates.UpdateAndExecute2[T0,T1,TRet](CallSite site, T0 arg0, T1 arg1)
   en Microsoft.Scripting.Interpreter.DynamicInstruction`3.Run(InterpretedFrame frame)
   en Microsoft.Scripting.Interpreter.Interpreter.Run(InterpretedFrame frame)
   en Microsoft.Scripting.Interpreter.LightLambda.Run4[T0,T1,T2,T3,TRet](T0 arg0, T1 arg1, T2 arg2, T3 arg3)
   en IronPython.Compiler.PythonCallTargets.OriginalCallTarget3(PythonFunction function, Object arg0, Object arg1, Object arg2)
   en CallSite.Target(Closure , CallSite , Object , Object , EventArgs )
   en System.Dynamic.UpdateDelegates.UpdateAndExecute3[T0,T1,T2,TRet](CallSite site, T0 arg0, T1 arg1, T2 arg2)
   en _Scripting_(Object[] , Object , EventArgs )
   en System.Windows.Forms.Control.OnClick(EventArgs e)
   en System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   en System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   en System.Windows.Forms.Control.WndProc(Message& m)
   en System.Windows.Forms.ButtonBase.WndProc(Message& m)
   en System.Windows.Forms.Button.WndProc(Message& m)
   en System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

Thank you very much!

**Comment by jmcouffin** [comment]
fix #2355
